packages/expo/src/errors/ExpoErrorManager.ts

```diff
@@ -0,0 +1,57 @@
+// Similar interface to the one used in expo modules.
+type CodedError = Error & { code?: string };
+
+let isErrorHandlingEnabled = true;
+let wasHit = false; // whether the original error handler was called
+
+const developmentBuildMessage = `If you're trying to use a module that is not supported in Expo Go, you can create a custom development build. See https://docs.expo.dev/development/introduction/ for more info.`;
```

Comment:
Suggestion:
- make clear this is a requirement
- normalize creating development builds by removing "custom"
- clarify it's a development build of the app at hand

```suggestion
const developmentBuildMessage = `If you're trying to use a module that is not supported in Expo Go, you need to create a development build of your app. See https://docs.expo.dev/development/introduction/ for more info.`;
```